otaigbe/Epic-Mail
# #164747539 Implementing Lfa's Feedback

#### What does this PR do?
Implement's some of the feedback from the LFA.
#### Description of Task to be completed?
- Sharpening up my UI
- Adding current badges
#### How should this be manually tested?
- For the UI open in a browser
- To observe the badges view the Readme file at the develop branch
#### What are the relevant pivotal tracker stories?
Pivotal tracker id #164747539
